packages/tabs/src/component-tabs.tsx

```diff
@@ -6,24 +6,23 @@ import { tabs as ccTabs, gridLayout } from '@warp-ds/css/component-classes';
 import type { TabsProps } from './props.js';
 import { debounce } from './utils.js';
 
-const setup = ({ className, children, onClick, active, ...rest }: any, tabsRef, wunderbarRef) => ({
-  nav: classNames(ccTabs.wrapperUnderlined, {
-    [className]: !!className,
-  }),
-  div: classNames(ccTabs.tabContainer, {
-    [gridLayout[`cols${children.filter((node) => node).length}`]]: true,
-  }),
-  wunderbar: classNames(ccTabs.wunderbar),
+const setup = (
+  { className, children, onClick, active, ...rest }: any,
+  tabsRef: RefObject<HTMLDivElement>,
+  selectionIndicatorRef: RefObject<HTMLDivElement>,
+) => ({
+  nav: classNames([!!className && className, ccTabs.wrapper]),
```

Comment:
No you're right, we don't need that check for `className`! 👍  If there is no `className` then it will just apply an empty string. I'll remove the check :)

Reply:
Updated in this commit: [1856b37](https://github.com/warp-ds/react/pull/267/commits/1856b37c0353131cc82dd67eff8664b47c27f08d)